src/Login/Login.js

```diff
@@ -3,35 +3,29 @@ import PropTypes from 'prop-types';
 import './Login.css';
 
 class Login extends Component {
-  runChange = (event) => {
+
+  runChange = async (event) => {
     event.preventDefault()
-    this.submitLogin()
-    this.props.action()
+    const user = await this.submitLogin()
+    this.props.action(user)
   }
 
-  submitLogin = () => {
-    const requestLogin = {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
+  submitLogin = async () => {
+  const response = await fetch(
+    "https://rancid-tomatillos.herokuapp.com/api/v2/login", {
+      method: "POST",
+      headers: {
+          "content-type": "application/json"
+      },
       body: JSON.stringify({
         email: document.getElementById('username-input').value,
         password: document.getElementById('password-input').value
```

Comment:
The username and password inputs should be using a controlled form to hold these values. Instead, pass in the username and password into the submitLogin function from wherever you call it.

Reply:
Additionally, the submitLogin function should be moved into your api helpers file.